centreon/packages/js-config/cypress/e2e/configuration.ts

```diff
@@ -50,16 +52,18 @@ export default ({
       },
       setupNodeEvents: async (cypressOn, config) => {
         const on = require('cypress-on-fix')(cypressOn)
-        installLogsPrinter(
-          on,
-          {
-            commandTrimLength: 5000,
-            defaultTrimLength: 5000,
-          }
-      );
+        installLogsPrinter(on, {
+          commandTrimLength: 5000,
+          defaultTrimLength: 5000,
+        });
         await esbuildPreprocessor(on, config);
         tasks(on);
 
+        cypressOn("task", {
+          fileExists(filePath: string) {
+            return fs.existsSync(filePath); // Checks if the file exists
+          },
+});
```

Comment:
move it to https://github.com/centreon/centreon/blob/develop/centreon/packages/js-config/cypress/e2e/tasks.ts